next/.eslintrc.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
         "plugin:@typescript-eslint/recommended-requiring-type-checking",
-        "plugin:@next/next/recommended",
+        "next",
```

Comment:
Before this change I was seeing this strange error; not sure why but it seems to fix it:

```
Oops! Something went wrong! :(

ESLint: 8.46.0

ESLint couldn't determine the plugin "@next/next" uniquely.

- /app/next/node_modules/@next/eslint-plugin-next/dist/index.js (loaded in ".eslintrc.js#overrides[0] » plugin:@next/next/recommended")
- /app/next/node_modules/eslint-config-next/node_modules/@next/eslint-plugin-next/dist/index.js (loaded in ".eslintrc.js » eslint-config-next » plugin:@next/next/recommended")

Please remove the "plugins" setting from either config or remove either plugin installation.
```